tests/plan.bats

```diff
@@ -228,15 +228,17 @@ setup() {
 }
 
 @test "plan/platform" {
+
    cd "$TESTDIR"
 
-   # Run with amd64 platform
-   run "$DAGGER" "do" -p./plan/platform/config_platform_linux_amd64.cue verify
+   # Run with invalid platform format
```

Comment:
Where are tests with real platforms?

Reply:
I didn't add tests will real platforms since it seemed somehow bloated to setup the CI system to install multiple platforms through emulation to test this. The only thing that we mostly care is that we're sending the right platform to the buildkit daemon and then let buildkit do it's thing. IMO we shouldn't be testing that buildkit is actually doing what it's supposed to do, that doesn't add a lot of value. Thoughts?